k8s/releases/discourse/discourse.yaml

```diff
@@ -1,121 +1,121 @@
+        DISCOURSE_AUTO_EMAIL_IN_DOMAIN: discourse.mozilla.org
+        DISCOURSE_AUTO_EMAIL_IN_ENABLED: true
+        DISCOURSE_CDN_URL: https://cdn.discourse-prod.itsre-apps.mozit.cloud
+        DISCOURSE_CONTENT_SECURITY_POLICY: false
+        DISCOURSE_EMAIL_IN: true
+        DISCOURSE_EMAIL_IN_AUTHSERV_ID: amazonses.com
+        DISCOURSE_ENABLE_CORS: true
+        DISCOURSE_ENABLE_LOCAL_LOGINS: false
+        DISCOURSE_ENABLE_S3_UPLOADS: true
+        DISCOURSE_FORCE_HTTPS: true
+        DISCOURSE_HOSTNAME: discourse.mozilla.org
+        DISCOURSE_LOG_SIDEKIQ: 1
+        DISCOURSE_LOG_SIDEKIQ_INTERVAL: 1
+        DISCOURSE_MANUAL_POLLING_ENABLED: true
+        DISCOURSE_NOTIFICATION_EMAIL: notifications@discourse.mozilla.org
+        DISCOURSE_REPLY_BY_EMAIL_ADDRESS: replies+%{reply_key}@discourse.mozilla.org
+        DISCOURSE_REPLY_BY_EMAIL_ENABLED: true
+        DISCOURSE_S3_REGION: us-west-2
+        DISCOURSE_S3_USE_IAM_PROFILE: true
+        DISCOURSE_SMTP_ADDRESS: email-smtp.us-west-2.amazonaws.com
+        DISCOURSE_SMTP_PORT: 587
+        PG_MAJOR: 12
+    db:
+      migrate:
+        enabled: true
+    deployment:
+      arn_role: arn:aws:iam::783633885093:role/discourse/discourse-prod
+    externalSecrets:
+      enabled: true
+      name: discourse
+      secrets:
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_AKISMET_API_KEY
+        property: DISCOURSE_AKISMET_API_KEY
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_AUTH0_CALLBACK_URL
+        property: DISCOURSE_AUTH0_CALLBACK_URL
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_AUTH0_CLIENT_ID
+        property: DISCOURSE_AUTH0_CLIENT_ID
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_AUTH0_DOMAIN
+        property: DISCOURSE_AUTH0_DOMAIN
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_AUTH0_CLIENT_SECRET
+        property: DISCOURSE_AUTH0_CLIENT_SECRET
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_DB_HOST
+        property: DISCOURSE_DB_HOST
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_DB_NAME
+        property: DISCOURSE_DB_NAME
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_DB_PASSWORD
+        property: DISCOURSE_DB_PASSWORD
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_DB_USERNAME
+        property: DISCOURSE_DB_USERNAME
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_REDIS_HOST
+        property: DISCOURSE_REDIS_HOST
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_S3_UPLOAD_BUCKET
+        property: DISCOURSE_S3_UPLOAD_BUCKET
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_SMTP_PASSWORD
+        property: DISCOURSE_SMTP_PASSWORD
+      - key: /prod/discourse/envvar
+        name: DISCOURSE_SMTP_USER_NAME
+        property: DISCOURSE_SMTP_USER_NAME
+    image:
+      pullPolicy: Always
+      repository: 783633885093.dkr.ecr.us-west-2.amazonaws.com/discourse
+      tag: prod-v1.0.0
```

Comment:
Looks like the biggest change outside of un-commenting. I saw you release this earlier. https://github.com/mozilla/discourse.mozilla.org/releases/tag/v1.0.0